src/lib/run_environment.ml

```diff
@@ -94,10 +94,37 @@ module Tool = struct
   end
 end
 
+module Make_fun = struct
+  module Requirement = struct
+    type t = [
+      | `Processors of int
+      | `Internet_access
+      | `Memory of [ `GB of float | `Decent | `Big ]
```

Comment:
yes they are “subjective and uninformative" but that's all we know so far.

That's why I said it's a first stab; all I “know” is that some jobs are “normal” (= decent) and some others are "big" (will fail with less than, say, 4 GB allocated to them and their size increases with data size).

"quick-run" means "if possible I'd prefer to be run on the login node of a cluster because the scheduling part is slower than the actual job"